scripts/ndt_build_and_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -x
+set -e
+
+# Use a clean directory, so that we can mount it from docker run and
+# have access to the finished product, when required.
+mkdir ~/ndt; cd ~/ndt; cp ~/builder/slice-tags.list .
```

Comment:
Just to double check -- the version in that file looks very old -- pre-websocket and TLS support. Is this intentional?